### ISSUE TITLE
Availability on F-Droid?

### ISSUE THREAD

**Issue by github-user-1** [ds10git/tvbrowserandroid#16]
I just came accross this app by chance and think it's interessting. As I get my apps from F-Droid and I unfortunately couldn't find it there... is there any chance to get it published there?

**Comment by monojp** [comment]
Yeah, according to https://f-droid.org/wiki/page/org.tvbrowser.tvbrowser the reason for not being on F-Droid is  #7

**Comment by DJCrashdummy** [comment]
maybe [an own fdroid-repo](https://f-droid.org/en/repomaker/) or linking to fdroid & [izzy's repo](https://forum.f-droid.org/t/known-repositories/721) (where tv-browser is available) would make sense...

**Comment by baimafeima** [comment]
I really hope that you can make it available on F-Droid. I don't know anyone who still uses that thing called Google.

**Comment by aslmx** [comment]
Would also like releases through f-droid 👍

Any progress? 🙃

**Comment by Krawei** [comment]
Dear devs,

is there any progress here? It would be great to have that gorgeous app on F-Droid!

Thanks dor your great work!

Kind Regards,
K.
